tests/nexus_test.py

```diff
@@ -289,9 +303,18 @@ def test_load_detector_select_entry_if_not_unique(nexus_file, expected_bank12):
 
 
 @pytest.mark.parametrize('entry_name', [None, nexus.NeXusEntryName('entry-001')])
-def test_load_monitor(nexus_file, expected_monitor, entry_name):
+@pytest.mark.parametrize(
+    'data_slice',
+    [
+        None,
+        ('time', slice(sc.datetime('1970-01-01T00:00:00', unit='ms'), None)),
+    ],
+)
+def test_load_monitor(nexus_file, expected_monitor, entry_name, data_slice):
+    data_slice = {'data_slice': data_slice} if data_slice is not None else {}
     monitor = nexus.load_monitor(
         nexus_file,
+        **data_slice,
         monitor_name=nexus.NeXusMonitorName('monitor'),
         entry_name=entry_name,
     )
```

Comment:
Why is the expected monitor the same in both cases? Looks like we are not testing whether the param is actually used?

Reply:
That's true. I'll fix that

Reply:
Not sure why you think it makes little sense? It should be easy to do?

Reply:
I just meant we don't really want to test that scippnexus does what it says it will, that is, that slicing the file and slicing the array gives the same result.

The only thing we really want to test here is that the slice is applied to the file.

Reply:
> The only thing we really want to test here is that the provided slice is applied to the file.

Indeed, and how do you check that?

Reply:
Before I checked it by looking for the dimension error when slicing by a dimension that doesn't exist.
Now I do like you suggested and compare the sliced loaded data and the sliced expected data.

Reply:
Note that a lot of things in Scipp raise `DimensionError`. Also, it does not tell you if you passed in indices correctly? 👍 for the update.